tox.ini

```diff
@@ -44,7 +44,7 @@ commands =
     casa: cp {toxinidir}/casa_config_template.py config.py
     casa: python -m casaconfig --update-all
     casa: python -m casaconfig --current-data
-    pip freeze
+    uv pip freeze
```

Comment:
@bsipocz - what about using ``{list_dependencies_command}`` instead, which should be more general?